docs/source/gcp/examples/network-cloudrouter.rst

```diff
@@ -0,0 +1,18 @@
+Network - Check if there are unattached Cloud Routers
+============================================================
```

Comment:
in general we should use the class documentation for reference examples, not every action/filter/resource needs an example in the folder, especially if its not a common usage scenario.

Reply:
I would say it's rather use case for cost savings/keeping a project clean and well structured than a reference example.

Reply:
again in the real world this is pretty corner case not common case. how often have you seen projects with extraneous cloud routers ?

Reply:
Fixed